code/game/objects/items/storage/book.dm

```diff
@@ -174,6 +174,12 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible", "
 			var/unholy2clean = A.reagents.get_reagent_amount("unholywater")
 			A.reagents.del_reagent("unholywater")
 			A.reagents.add_reagent("holywater",unholy2clean)
+		if(istype(A, /obj/item/storage/book/bible) && !istype(A, /obj/item/storage/book/bible/syndicate))
+			to_chat(user, "<span class='notice'>You purify [A], conforming it to your belief.</span>")
```

Comment:
Wouldnt you be able to spam this by clicking on the same bible over and over?

Reply:
yeah and spam your own chatbox

Reply:
It only tells the person doing it, so the only one they can spam is themself

Reply:
I got that, I guess my point was that you shouldn't be able to do this at all. Maybe add a check to see if the icon state, name, and item_state are the same?

Reply:
There's like no impact, and there's like 30 different things that you can spam yourself with